src/governance-object.h

```diff
@@ -25,8 +25,8 @@ class CGovernanceObject;
 class CGovernanceVote;
 
 static const int MAX_GOVERNANCE_OBJECT_DATA_SIZE = 16 * 1024;
-static const int MIN_GOVERNANCE_PEER_PROTO_VERSION = 70206;
-static const int GOVERNANCE_FILTER_PROTO_VERSION = 70206;
+static const int MIN_GOVERNANCE_PEER_PROTO_VERSION = 70208;
+static const int GOVERNANCE_FILTER_PROTO_VERSION = 70208;
```

Comment:
The meaning of the `GOVERNANCE_FILTER_PROTO_VERSION` is `at which protocol version governance filter functionality was introduced` (similar to other versions in version.h) and it should never be bumped.